CHANGELOG-4.0.md

```diff
@@ -12,6 +12,10 @@
 - Fixed `Phalcon\Collection\Exception` to extend Phalcon\Exception
 - Fixed `Phalcon\Storage\Adapter\AbstractAdapter::initSerializer` to throw exception if `null === $this->serializerFactory && null === $this->serializer` [#14324](https://github.com/phalcon/cphalcon/issues/14324)
 
+## Fixed
```

Comment:
Can you remove this header and add the bulletpoint to the rest of the list above?